documentation/synthetic_tokens/security/security.md

```diff
@@ -0,0 +1,107 @@
+This results in the contract GCR drastically increasing (~100x), making all later sponsors required to be massively over collateralized. 
+At best, this slows down future token sponsorship (must repeatedly create and wait on withdraw), and at worst, this can lock all future sponsorship, constricting token supply. 
+This could make it difficult or impossible for liquidators to liquidate positions and simultaneously drive up the token price.
+One worrisome attack vector is if a sponsor were to use this to constrict token supply, and then issue a withdraw request to bring their collateralization ratio to 0%. 
+If liquidators cannot get tokens to liquidate the sponsor’s position during the liveness period, the sponsor could walk away with a profit. 
+There are a number of assumptions that have to be in place for this to work: 
+1. Few or no tokens were minted before this sponsor increased the GCR
+2. The sponsor is better capitalized than any liquidator
+3. A liquidator does not have enough tokens or collateral to issue a partial liquidation large enough to wipe out the profitability of the sponsor. 
+If the sponsor is 100x collateralized, a partial liquidation for 1/100ths of the sponsor’s tokens would essentially wipe out all of the profit.
+Any larger liquidations, and they would lose money after the remaining withdrawal went through. 
+
+## Open issues affecting accurate fee payments:
+### Fees may be charged on collateral outside of “Profit From Corruption”
+* Scenario: In the current implementation, collateral that isn’t part of PFC is still taxed, which technically isn’t correct. Once a liquidation is settled, any collateral that hasn’t been withdrawn is charged until withdrawn. Similarly, once the contract expires, all non-withdrawn sponsor and tokenholder collateral is charged until withdrawn. 
+* Solution: It is up to token holders and liquidators to withdraw collateral they are owed in a timely manner. 
+
+### Final fee for the DVM is determined at liquidation time, not dispute time
+* Scenario: The final fee is queried at the time of liquidation, which is not when the price request is issued to the DVM. 
+If the final fee were to change between the liquidation and dispute, the contract could technically pay a different final fee than requested by the DVM.
+* Solution: The DVM rules should include some flexibility around exactly when fees are queried vs paid. 
+The liveness period is generally quite short, so this flexibility doesn’t add much risk.
+
+## Open issues affecting ability to withdraw collateral:
+### Network congestion
+* Scenario: UMA’s liquidation or dispute bots are down for longer the liveness periods for  liquidations and disputes. 
+This causes token sponsors to potentially be improperly liquidated or for the system to become undercollateralized. 
+* Solution: Robust monitoring of liquidator and dispute bots. Increase the liveness period parameter.
```

Comment:
These aren't complete sentences, which is out of place with the rest of the doc.